src/OpenTelemetry/BatchExportProcessor.cs

```diff
@@ -69,6 +71,7 @@ public abstract class BatchExportProcessor<T> : BaseExportProcessor<T>
             Guard.ThrowIfOutOfRange(exporterTimeoutMilliseconds, min: 0);
 
             this.circularBuffer = new CircularBuffer<T>(maxQueueSize);
+            this.circularBufferDroppedCount = Sdk.SdkInternalStateMeter.CreateObservableCounter($"{typeof(T).Name}.BatchExportProcessorDroppedCount", () => this.DroppedCount);
```

Comment:
Use the exporter name here instead of `typeof(T)`.

Reply:
I kind of like the type of item as the name, could exporter name be a dimension?

Reply:
> I kind of like the type of item as the name, could exporter name/type be a dimension?

+1